### PR TITLE
Rename GoogleReader to OpenReader and some small changes

### PR DESCRIPTION
**Pull request by josh64x2**
- Renamed `GoogleReader` to `OpenReader`
- Tidied up some code for Swift compatibility

### CODE REVIEW

File: src/OpenReader.m [hunk 1]
```diff
@@ -1085,4 +1083,14 @@ - (void)createFolders:(NSMutableArray *)params
...
 }
 
+
+/**
+ Get the current timestamp
+ 
+ @return current timestamp as a string
+ */
++ (NSString *)currentTimestamp {
+    return [NSString stringWithFormat:@"%0.0f",[[NSDate date] timeIntervalSince1970]];
```
Comment (Eitot) [inline src/OpenReader.m]: `timeIntervalSince1970` is a property. Use dot notation here.

File: src/OpenReader.m [hunk 2]
```diff
@@ -1085,4 +1083,14 @@ - (void)createFolders:(NSMutableArray *)params
     }
 }
 
+
+/**
+ Get the current timestamp
+ 
+ @return current timestamp as a string
+ */
++ (NSString *)currentTimestamp {
```
Comment (Eitot) [inline src/OpenReader.m]: Back this by a private, read-only property.
```objc
@property (readonly, nonatomic) NSString *currentTimestamp;
```

File: src/OpenReader.m [hunk 3]
```diff
@@ -480,15 +478,15 @@ -(ASIHTTPRequest*)refreshFeed:(Folder*)thisFolder withLog:(ActivityItem *)aItem
     }
 		
 	NSURL *refreshFeedUrl = [NSURL URLWithString:[NSString stringWithFormat:@"%@stream/contents/feed/%@?client=%@&comments=false&likes=false%@&ck=%@&output=json",APIBaseURL,
-                                                  percentEscape(feedIdentifier),ClientName,itemsLimitation,TIMESTAMP]];
+                                                  percentEscape(feedIdentifier),ClientName,itemsLimitation,[OpenReader currentTimestamp]]];
```
Comment (Eitot) [inline src/OpenReader.m]: Use dot notation here.

Comment (josh64x2) [inline src/OpenReader.m]: Unfortunately you are not meant to use dot notation on class methods. It does work, but xcode will not auto complete and it is frowned upon.

File: src/OpenReader.m (hunk 2 above)
Comment (josh64x2) [inline src/OpenReader.m]: As before, this is a class method and my understanding is properties should only be used for instance variables - at least thats what I think [Apple's document is telling me](https://developer.apple.com/library/content/releasenotes/ObjectiveC/ModernizationObjC/AdoptingModernObjective-C/AdoptingModernObjective-C.html#//apple_ref/doc/uid/TP40014150-CH1-SW4) 😬

Comment (josh64x2) [inline src/OpenReader.m]: I am happy to consider other advice like changing to instance method or if there is some document that says it is ok to use read only properties with class methods in the case of a singleton etc 👍

Comment (Eitot) [inline src/OpenReader.m]: The document is a bit outdated. You can declare class properties too:
```objc
@property (readonly, class, nonatomic) NSString *currentTimestamp;
```

Then they are implemented by a class method as a getter. Dot notation offers better semantics and Apple has been using it everywhere for accessors. Whether they are backed by an ivar is an implementation detail.

**Pull request merged**